spec/declaration.dd

```diff
@@ -55,6 +55,12 @@ $(GNAME DeclaratorIdentifierList):
 $(GNAME DeclaratorIdentifier):$(LEGACY_LNAME2 VarDeclaratorIdentifier)
     $(GLINK_LEX Identifier)
     $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters)$(OPT) $(D =) $(GLINK Initializer)
+    $(GLINK BitfieldDeclarator)
+
+$(GNAME BitfieldDeclarator):
+    $(D :) $(GLINK2 expression, AssignExpression)
+    $(GLINK_LEX Identifier) $(D :) $(GLINK2 expression, AssignExpression)
+    $(GLINK_LEX Identifier) $(D :) $(GLINK2 expression, AssignExpression) $(D =) $(GLINK Initializer)
```

Comment:
Using AssignExpression after `:` introduces an ambiguity for the following example:
`
int a : b = c;
`

The part `b = c` could be an AssignExpression or `b` could be an AssignExpression and `c` an Initializer.

The parser seems to use a ConditionalExpression instead: https://github.com/dlang/dmd/blob/df3b0559c21b65ebf62f508f100fc4e25ffd4885/src/dmd/parse.d#L4614

Reply:
Q: Initial implementation [forbids](https://github.com/dlang/dmd/pull/13568/files#diff-7b6e6126b5e5b6f47c0389d7533a9336a434356c4b2c515fc43bb44328a7123aR4) bitfield initializers. Does this mean that they will be supported in the future?

Reply:
Update for @tim-dlang's observation:
```suggestion
    $(GLINK_LEX Identifier) $(D :) $(GLINK2 expression, ConditionalExpression) $(D =) $(GLINK Initializer)
```

Reply:
It would be a reasonable enhancement.

Reply:
@tim-dlang you're right about the ConditionalExpression.